src/signals/incident/definitions/wizard-step-2-vulaan.js

```
@@ -8,310 +13,31 @@ export default {
-          ifAllOf: {
-            subcategory: 'overlast-op-het-water-snel-varen'
-          }
-        },
-        render: FormComponents.TextInput
-      },
-      extra_boten_snelheid_meer: {
-        meta: {
-          label: 'Zijn er nog dingen die u ons nog meer kunt vertellen?',
-          subtitle: 'Bijvoorbeeld: de kleur(en) van de boot, het aantal passagiers, de vaarrichting, Y of Vignet nummer etc.',
-          pathMerge: 'extra_properties',
-          ifAllOf: {
-            subcategory: 'overlast-op-het-water-snel-varen'
-          }
-        },
-        render: FormComponents.TextareaInput
-      },
-      extra_boten_geluid_meer: {
-        meta: {
-          label: 'Zijn er nog dingen die u ons nog meer kunt vertellen?',
-          pathMerge: 'extra_properties',
-          ifAllOf: {
-            subcategory: 'overlast-op-het-water-geluid'
-          }
-        },
-        render: FormComponents.TextareaInput
-      },
-      extra_boten_gezonken_meer: {
-        meta: {
-          label: 'Zijn er nog dingen die u ons nog meer kunt vertellen?',
-          subtitle: 'Bijvoorbeeld: "er lekt olie", "gevaar voor andere boten", etc.',
-          pathMerge: 'extra_properties',
-          ifAllOf: {
-            subcategory: 'overlast-op-het-water-gezonken-boot'
-          }
-        },
-        render: FormComponents.TextareaInput
-      },
-      navigation_submit_button: {
-        meta: {
-          ifNoneOf: {
-            subcategory: [
-              'straatverlichting-openbare-klok',
-              'klok'
-            ]
-          }
-        }
-      },
-      $field_0: {
-        isStatic: false,
-        render: IncidentNavigation
-      }
+      case 'overlast-van-dieren':
+        return overlastVanDieren;
+
+      case 'overlast-van-en-door-personen-of-groepen':
+        return overlastPersonenEnGroepen;
+
+      case 'wegen-verkeer-straatmeubilair':
+        return wegenVerkeerStraatmeubilair;
+
+      default:
+        return {};
```

This shouldn't happen right? Probably we should just throw an error instead of failing silently.

---

als je een hoofdcategorie gebruikt die niet bestaat wel